README.md

```diff
@@ -1,18 +1,31 @@
-# simpler_eda 
+# A package for Simpler Explanatory Data Analysis (simpler_eda)
 
 ![](https://github.com/nichowu/simpler_eda/workflows/build/badge.svg) [![codecov](https://codecov.io/gh/nichowu/simpler_eda/branch/main/graph/badge.svg)](https://codecov.io/gh/nichowu/simpler_eda) ![Release](https://github.com/nichowu/simpler_eda/workflows/Release/badge.svg) [![Documentation Status](https://readthedocs.org/projects/simpler_eda/badge/?version=latest)](https://simpler_eda.readthedocs.io/en/latest/?badge=latest)
 
-Python package that eases the pain of plotting numeric and categorical features in Altair
+
+## Overview
+
+Exploratory Data analysis (EDA) is an important step in any data analysis.  However, carrying out EDA with the Altair package requires a lot of coding effort. Moreover, it assumes a basic knowledge of functions and grammar of graphics syntax that are appropriate for visualizing categorical and numerical variables. The simpler_eda package addresses this issue by providing functions that are tailored to produce categorical, numerical and correlation plots using a single line of code. Furthermore, the package provides customization capability for the plots based on specific user needs (theme, title, font, size and etc.). The users are able to spend more time on analyzing the data set and less time configuring Altair plot settings.
 
 ## Installation
 
 ```bash
 $ pip install -i https://test.pypi.org/simple/ simpler_eda
 ```
 
-## Features
+## Functions
 
-- TODO
+This package contains three functions, each that accepts a pandas `DataFrame` for EDA. The EDA functions can be used with a dataset with numerical and categorical features. Each functions will have it's own required and optional arguments to configure the properties of the plot. 
+
+1. `correlation_eda_plot:`  Plot a correlation map with the given dataframe object and a list of numerical features. Users are allow to set multiple arguments regarding the setting of the correlation plot from color schemes, plot width, height and plot title.
```

Comment:
Users are **_allowed_** to set multiple arguments regarding the setting of the correlation plot **_including_** color schemes, plot width, height and plot title.

Reply:
Good observation!